242.cs

```diff
@@ -0,0 +1,16 @@
+public class Solution {    
+    public bool IsAnagram (string s, string t){
+        if (s.Length != t.Length) return false;
+        char[] lettersOf_S = s.ToCharArray();
+        Array.Sort(lettersOf_S);
+        char[] lettersOf_T = t.ToCharArray();
+        Array.Sort(lettersOf_T);
+        return IsAnagramVerified(lettersOf_S,lettersOf_T);
+    }
+
+    private bool IsAnagramVerified(char[] s, char[] t, int i = 0, int j = 0){
```

Comment:
There is no need to make this method recursive, that will degrade the performance of the algorithm and doesn't make the code more readable

Reply:
tabien